types/mparticle__web-sdk/index.d.ts

```diff
@@ -3,6 +3,45 @@ import { Batch } from "@mparticle/event-models";
 // Placeholder for Dictionary-like Types
 export type Dictionary<V = any> = Record<string, V>;
 
+// Rokt Manager Types
+export interface RoktPartnerAttributes {
+    [key: string]: string | number | boolean | undefined | null;
+}
+
+export interface RoktPartnerExtensionData<T> {
+    [extensionName: string]: T;
+}
+
+export interface RoktSelectPlacementsOptions {
+    attributes: RoktPartnerAttributes;
+    identifier?: string;
+}
+
+export type RoktPlacement = Record<string, unknown>;
+
+export interface RoktSelection {
+    close: () => void;
+    getPlacements: () => Promise<RoktPlacement[]>;
+}
```

Comment:
This is partially there. I think we should capture the full type - https://github.com/ROKT/sdk-web/blob/main/packages/web-sdk/controller/src/controller/platform/services/selections/selection.ts#L30-L41